react-accordion-component/app-name/src/Accordion.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+export default function Accordion({ topics }) {
+  const [topicClickedIndex, setTopicClickedIndex] = useState(undefined);
+  const accordion = topics.map((topic, index) => (
+    <div key={index} className="topic">
+      <div
+        className="header"
+        onClick={() =>
+          topicClickedIndex === index
+            ? setTopicClickedIndex(undefined)
+            : setTopicClickedIndex(index)
+        }>
+        <Topic subject={topic.title} topicIndex={index} />
+      </div>
+      <div
+        className="info"
+        style={{ display: topicClickedIndex === index ? 'flex' : 'none' }}>
+        <Info info={topic.content} />
+      </div>
```

Comment:
This is great!
An alternative to using CSS `display` to show and hide the content is to use conditional rendering. Something like:
```
{ topicClickedIndex === index && <Info info={topic.content} /> }
```

With CSS, the item is still in the DOM. With conditional rendering it is removed from the DOM completely and put back in.

Using CSS and `display` is a good approach if you want to do animation. If you don't care about animation, conditional rendering can be simpler and more efficient.